ll_insertions/ll_insertions.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const Node = require;
+
+module.exports = class LinkedList {
+  constructor() {
+    this.head = null;
+  }
+  
+  append(value) {
+    const node = new Node(value);
+    
+    if (!this.head) {
+      this.head = node;
+      return this;
+    }
+    let currentNode = this.head;
+    while (currentNode.next) {
+      currentNode = currentNode.next;
+    }
+    currentNode.next = node;
+    return this;
+  }
+  
+  insertBefore(value, newValue) {
```

Comment:
Good job on this method. But what happens if the value that we're trying to find is not in the linked list?